### PR TITLE
165 verification validation

### LINKED ISSUE #165: Verification & Validation
Verify, simulate and test that the implemented system fulfils the requirements.

### PR DESCRIPTION
**Pull request by Joergenson**

### CODE REVIEW

File: project_report/tex/verification_and_validation.tex [hunk 1]
```diff
@@ -1 +1,61 @@
 \section{Verification \& Validation}
+This section describes the verification and validation of the developed MES.\\
+
+Verification ensures the quality of the software, design, architecture and the
+like by checking if the software has been build according to the plan.
```
Comment (Bokaiii) [inline project_report/tex/verification_and_validation.tex]: build = built

File: project_report/tex/verification_and_validation.tex [hunk 2]
```diff
@@ -1 +1,61 @@
...
+Validation is testing if the developed system meets the brewery's requirements
+found in Table \ref{table:Requirements}.
+
+\subsection{Verification}
+The source code is compared to the results of the architecture and design
+sections to verify the developed system. \\
+
+When comparing, the focus is on ensuring that the system complies with the
+design principle separation of concerns. The database should follow the
```
Comment (Bokaiii) [inline project_report/tex/verification_and_validation.tex]: Comma after principle

File: project_report/tex/verification_and_validation.tex [hunk 3]
```diff
@@ -1 +1,61 @@
...
+and the OPC-UA client should communicate via HTTP. \\
+
+The database can store relevant data, and the data corresponds with the tables
+specified in the EER diagram. The dashboard follows the MVC pattern, as the
+Model controls the data-related logic, the View controls the user interface
+logic, and the Controller controls the business logic. The OPC-UA and the REST
+API are separate subsystems that communicate through the HTTP protocol. 
+
+
+\subsection{Validation}
```
Comment (Bokaiii) [inline project_report/tex/verification_and_validation.tex:25]: Should be rewritten in present tense

File: project_report/tex/verification_and_validation.tex [hunk 4]
```diff
@@ -1 +1,61 @@
...
+The validation tests consisted of producing every product type at different
+machine speeds with the same number of products to produce and gathering the
```
Comment (Bokaiii) [inline project_report/tex/verification_and_validation.tex]: Gathering = gather

File: project_report/tex/verification_and_validation.tex [hunk 5]
```diff
@@ -1 +1,61 @@
...
+production data. These tests made it possible to calculate the OEE for every
+speed and then the error function for the product types.\\
+
+The validations below were concluded during the test and are listing the
```
Comment (Bokaiii) [inline project_report/tex/verification_and_validation.tex]: Is test supposed to be singular? In line 26 you mention tests (plural)

File: project_report/tex/verification_and_validation.tex [hunk 6]
```diff
@@ -1 +1,61 @@
...
+usability.
+
+\subsubsection{Control production} 
+The system allows control of the production line. The production manager can
+start, stop, clear, reset and abort the production through the dashboard. This
+functionality fulfils requirements R01-R02 in the list of requirements.
+
+\subsubsection{Data processing}
+The system monitors batch data from the production line. The batch data is
+stored in the database and kept track by using the batch id. This functionality
```
Comment (Bokaiii) [inline project_report/tex/verification_and_validation.tex]: ...database and kept track OF, USING the batch id.

File: project_report/tex/verification_and_validation.tex [hunk 7]
```diff
@@ -1 +1,61 @@
...
+allows the data to be accessed in the history page of the dashboard after it has
+run. Thus fulfilling requirements R03-R06.
```
Comment (Bokaiii) [inline project_report/tex/verification_and_validation.tex]: Replace "." after run with comma (remember lower case t in thus

**Review by Bokaiii — CHANGES_REQUESTED** [review]

**Review by kkmp-dk — APPROVED** [review]

**Review by Bokaiii — APPROVED** [review]

**Pull request merged**